### ISSUE TITLE
Nice. Thanks for sharing

### ISSUE THREAD

**Issue by neilyoung** [givo/lib-kurento#1]
However, can't use your lib in my node ts app. What might go wrong?

`import * as libKurento from 'lib-kurento';
`

src/index.ts(2,29): error TS2307: Cannot find module 'lib-kurento'.

**Comment by neilyoung** [comment]
OK, got it. But while using gulp on your lib i had some errors:

```
~/Documents/Kurento/lib-kurento $ gulp
[10:36:09] Using gulpfile ~/Documents/Kurento/lib-kurento/gulpfile.js
[10:36:09] Starting 'scripts'...
[10:36:09] Finished 'scripts' after 17 ms
[10:36:09] Starting 'watch'...
[10:36:09] Finished 'watch' after 12 ms
[10:36:09] Starting 'default'...
gulp out
[10:36:09] Finished 'default' after 66 μs
src/endpoints/recorder/recorder-endpoint.interface.ts(3,18): error TS2430: Interface 'IRecorderEndpointWrapper' incorrectly extends interface 'IEndpointWrapper'.
  Types of property 'on' are incompatible.
    Type '{ (event: "RecordingStarted", listener: (event: any) => void): this; (event: "RecordingStopped", listener: (event: any) => void): this; }' is not assignable to type '{ (event: "MediaFlowingIn", listener: (event: any) => void): this; (event: "MediaStoppedFlowingIn", listener: (event: any) => void): this; (event: "MediaFlowingOut", listener: (event: any) => void): this; (event: "MediaStoppedFlowingOut", listener: (event: any) => void): this; }'.
      Types of parameters 'event' and 'event' are incompatible.
        Type '"MediaFlowingIn"' is not assignable to type '"RecordingStarted"'.
src/endpoints/webrtc/webrtc-endpoint.interface.ts(3,18): error TS2430: Interface 'IWebRTCEndpointWrapper' incorrectly extends interface 'IEndpointWrapper'.
  Types of property 'on' are incompatible.
    Type '(event: "ServerIceCandidate", listener: (candidate: any) => void) => this' is not assignable to type '{ (event: "MediaFlowingIn", listener: (event: any) => void): this; (event: "MediaStoppedFlowingIn", listener: (event: any) => void): this; (event: "MediaFlowingOut", listener: (event: any) => void): this; (event: "MediaStoppedFlowingOut", listener: (event: any) => void): this; }'.
      Types of parameters 'event' and 'event' are incompatible.
        Type '"MediaFlowingIn"' is not assignable to type '"ServerIceCandidate"'.
[10:36:11] TypeScript: 2 semantic errors
[10:36:11] TypeScript: emit succeeded (with errors)
^C
~
```

**Comment by givo** [comment]
Hi! 

I am aware to those errors, but don't worry they are only semantic errors.

**Comment by neilyoung** [comment]
@givo Thanks. I still had problems, but since it was of the "cannot find li-kurento" as result of my import, it is most likely my fault :) Thanks

**Comment by givo** [comment]
Great! 

I am going to close the issue, if you have any problems just let me know :)

**Comment by SongLin123** [comment]
@givo I have the same problem as you！

**Comment by givo** [comment]
@SongLin123 

The source of the problem is having an interface that inherits from another interface and trys to declares the same funcion on()

**Comment by SongLin123** [comment]
@givo
 Sorry I have no idea of this problem.
But,maybe change package.json `main:"./dist/index.ts"` =>`main:"./dist/index.js` can solve your ` Cannot find module 'lib-kurento'.` problem:)

**Comment by givo** [comment]
@SongLin123 

Fixed  in [v0.0.2](https://github.com/givo/lib-kurento/releases/tag/v0.0.2)
